docs/65_dandi_hub.md

```diff
@@ -0,0 +1,6 @@
+The DANDI ecosystem includes a self-hosted Jupyter notebook service. This service is orchestrated on a Kubernetes (k8s) cluster
+that provides different instance types of users to efficiently interact with data in the DANDI Archive.
+
+[Proceed to the following README](https://github.com/dandi/dandi-hub/blob/main/README.md#dandihub) to see how you can
+set up your own DANDI Hub -- **Note: it is important that your k8s cluster is in the same region
+as your data**
```

Comment:
Similar to your Google Doc, perhaps we can add some quick links to each page.

```suggestion
as your data**

Resources
1. [Source code and instructions]( https://github.com/dandi/dandi-hub)
1. [DANDI Hub](https://hub.dandiarchive.org/)
1. [LINC Hub](https://hub.lincbrain.org/)
```

Reply:
Thanks Kabi -- added.